patient/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+from .models import Patient
+
+class PatientSerializer(serializers.Serializer):
+    class Meta:
+        model = Patient
+        fields = '__all__'
+        read_only_fields = ('user_id',)

```

Comment:
What is use of `read_only_fields` ? Gets created once and cannot be updated?

Reply:
I am assuming cannot update user_id  but other information